KTLibrary/FormattedEncoder.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+from __future__ import annotations
+
+import json
+
+
+# Custom encoder class that inherits from json.JSONEncoder
+class FormattedEncoder(json.JSONEncoder):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.indentation_level = 0
+
+    def encode(self, o, force=False):
+        # Overwrites the encode function of the JSONEncoder by a custom one.
```

Comment:
Het is waarschijnlijk best om dit commentaar als een docstring met """triple double quotes""" te schrijven i.p.v. met #'s.